src/tree/GitHubConfigGroupTreeItem.ts

```diff
@@ -21,19 +23,44 @@ export type BuildConfigs = {
     'app_artifact_location'?: string
 }
 
+function getRangeFromError(error: YAMLSyntaxError): Range {
+    if (error.linePos) {
+        const { start, end } = error.linePos;
+        return new Range(start.line - 1, start.col - 1, end.line - 1, end.col - 1);
+    }
+    return new Range(0, 0, 0, 0);
+}
+
+function getYamlErrorMessage(error: unknown): string {
+    if (error instanceof YAMLSyntaxError) {
+        const range = getRangeFromError(error);
+        return `Error: Invalid YAML between lines ${range.start.line} and ${range.end.line}`;
+    } else {
+        return parseError(error).message;
+    }
+}
+
 export class GitHubConfigGroupTreeItem extends AzExtParentTreeItem {
     public static contextValue: string = 'azureStaticGitHubConfigGroup';
     public contextValue: string = GitHubConfigGroupTreeItem.contextValue;
-    public readonly label: string = localize('gitHubConfig', 'GitHub Configuration');
     public parent: EnvironmentTreeItem;
     public yamlFilePath: string;
     public buildConfigs: BuildConfigs | undefined;
 
-    public constructor(parent: EnvironmentTreeItem, yamlFilePath: string, buildConfigs: BuildConfigs) {
+    public constructor(parent: EnvironmentTreeItem, yamlFilePath: string, buildConfigs: BuildConfigs, public parseYamlError?: unknown) {
         super(parent);
         this.yamlFilePath = yamlFilePath;
         this.buildConfigs = buildConfigs;
         this.id = `${parent.id}-${this.yamlFilePath}`;
+        this.commandArgs = [this];
```

Comment:
I don't think this is necessary to fill out
> /**
     * The arguments to pass in when executing `commandId`. If not specified, this tree item will be used.
     */